Change the Export Multiple Grids demo description (#1781)
DevExpress/devextreme-demos: JSDemos/Demos/DataGrid/PDFExportMultipleGrids/description.md

> This demo shows two DataGrids in different tabs. The export system arranges these DataGrids on different pages of the PDF document. 

Reviewer: ```suggestion
This demo shows two DataGrids in different tabs. Click the "Export" button to export two grids and arrange them on different pages of the PDF document. 
```

> This demo shows two DataGrids in different tabs. Click the "Export" button to export two grids and arrange them on different pages of the PDF document. 

Author: ```suggestion
This demo shows two DataGrids in different tabs. Click the **Export multiple grids** button to export two grids and arrange them on different pages of the PDF document. 
```

> To implement this functionality, execute a <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then" target="_blank">chain of Promises</a>. Call the [exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/pdfExporter/#exportDataGridoptions) methods in a chain one after another. Use the **jsPDF**.<a href="https://raw.githack.com/parallax/jsPDF/master/docs/jsPDF.html#addPage" target="_blank">addPage()</a> method to add a page in the PDF document.

Reviewer: ```suggestion
To implement this functionality, call the [exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/pdfExporter/#exportDataGridoptions) methods in a <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then" target="_blank">chain of Promises</a> one after another. Use the **jsPDF**.<a href="https://raw.githack.com/parallax/jsPDF/master/docs/jsPDF.html#addPage" target="_blank">addPage()</a> method to add a page in the PDF document.
```

> To implement this functionality, call the [exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/pdfExporter/#exportDataGridoptions) methods in a <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then" target="_blank">chain of Promises</a> one after another. Use the **jsPDF**.<a href="https://raw.githack.com/parallax/jsPDF/master/docs/jsPDF.html#addPage" target="_blank">addPage()</a> method to add a page in the PDF document.

Reviewer 2: ```suggestion
To implement this functionality, call the [exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/pdfExporter/#exportDataGridoptions) methods in a <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then" target="_blank">chain of Promises</a>, one after another. Use the **jsPDF**.<a href="https://raw.githack.com/parallax/jsPDF/master/docs/jsPDF.html#addPage" target="_blank">addPage()</a> method to add a page in the PDF document.
```

> You can use the [customizeCell](/Documentation/ApiReference/Common/Object_Structures/PdfExportDataGridProps/#customizeCell) function to customize cell appearance after export. In this demo, the **customizeCell** function sets the alternating backgrounds for rows.

Reviewer: ```suggestion
You can use the [customizeCell](/Documentation/ApiReference/Common/Object_Structures/PdfExportDataGridProps/#customizeCell) function to customize cell appearance in the PDF document. In this demo, the **customizeCell** function highlights alternate (even) rows.
```
